signal/events_wakuv2.go

```diff
@@ -0,0 +1,12 @@
+package signal
+
+const (
+	// EventPeerStats is sent when peer is added or removed.
+	// it will be a map with capability=peer count k/v's.
+	EventPeerStats = "wakuv2.peerstats"
+)
+
+// SendPeerStats sends discovery.summary signal.
+func SendPeerStats(peerStats interface{}) {
+	send(EventPeerStats, peerStats)
```

Comment:
Is `peerStats` translated to JSON? I was checking  the definition for ConnStatus which I think is the data sent
```
type ConnStatus struct {
	IsOnline   bool
	HasHistory bool
}
```
and i did not see that the attributes had metadata like `json:"nameOfTheField,omitempty"``.  
Is this done automatically if that info is missing?

Reply:
Good catch! I missed those annotations. But it's translated fine - in status-react i receive the JSON fine.